rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSet.java

```diff
@@ -262,10 +262,10 @@
                                                                           .opt("lj")
                                                                           .longOpt("listjobs")
                                                                           .description("Retrieve a list of jobs managed by the Scheduler.If specified the latest 'x' number of jobs or 'z' maximum number of jobs starting from 'y'th job.") //Example: to return the five last jobs, >./proactive-client -lj limit=5. To return the list of z jobs from index y, >./proactive-client -lj from=y limit=z ")
```

Comment:
Use latest=x to filter the x latest jobs, or from=y limit=z to retrieve the z latest jobs from the yth.